win32ss/user/ntuser/desktop.c

```diff
@@ -28,7 +28,11 @@ IntFreeDesktopHeap(IN PDESKTOP pdesk);
 /* GLOBALS *******************************************************************/
 
 /* These can be changed via CSRSS startup, these are defaults */
+#ifdef _WIN64 // Workaround for CORE-19563 (there is probably a class leak)
```

Comment:
```suggestion
#ifdef _WIN64 // HACK: Workaround for CORE-19563 (there is probably a class leak)
```
or
```suggestion
#ifdef _WIN64 // FIXME: Workaround for CORE-19563 (there is probably a class leak)
```
That way if someone greps for "hack" or "fixme" they will find it.